components/prism-parser.js

```diff
@@ -1,67 +1,73 @@
-Prism.languages.parser = Prism.languages.extend('markup', {
-	'keyword': {
-		pattern: /(^|[^^])(?:\^(?:case|eval|for|if|switch|throw)\b|@(?:BASE|CLASS|GET(?:_DEFAULT)?|OPTIONS|SET_DEFAULT|USE)\b)/,
-		lookbehind: true
-	},
-	'variable': {
-		pattern: /(^|[^^])\B\$(?:\w+|(?=[.{]))(?:(?:\.|::?)\w+)*(?:\.|::?)?/,
-		lookbehind: true,
-		inside: {
-			'punctuation': /\.|:+/
+(function (Prism) {
+
+	var parser = Prism.languages.parser = Prism.languages.extend('markup', {
```

Comment:
I think we could update all of the initial assignments like this:

```js
var language = Prism.languages.language = { ... object ... };
```

which would be a little nicer than having it on a separate line.

Reply:
The whole idea of these assignments was that we do not have to write the whole name again, but I agree that your idea seems nicer.

Like this?
```js
Prism.languages.language = { ... object ... };

var language = Prism.languages.language;

Reply:
No, I just mean to suggest, when we want to also assign it to a separate variable, do it on one line, rather than two.